### PR TITLE
Changed info point to health

### PR DESCRIPTION
**Pull request by anilkusaksiz**
It turned out only /health endpoint works for WebSphere, so here is another change.
Tried on dbs-cx6-editorial Tomcat and microservices WebSphere

### CODE REVIEW

File: jenkins/bb-fuel-minimal.groovy [hunk 1]
```diff
@@ -32,7 +32,7 @@ pipeline {
         string(name: 'IDENTITY_CLIENT', defaultValue: 'hybrid-flow', description: 'Identity client')
         string(name: 'BB_FUEL_VERSION', defaultValue: 'latest', description: '')
         booleanParam(name: 'PRERELEASE', defaultValue: false, description: 'Only applicable if BB_FUEL_VERSION = latest')
-        booleanParam(name: 'HEALTHCHECK_USE_ACTUATOR', defaultValue: true, description: 'Healthcheck endpoint is changed since 2.17.0 release')
+        booleanParam(name: 'HEALTHCHECK_USE_ACTUATOR', defaultValue: true, description: 'Should be true if BOM version => 2.17.0')
```
Comment (richard-blank) [inline jenkins/bb-fuel-minimal.groovy]: true is default, having a message `"Should be false if DBS version is before 2.17.0"` makes more sense here

File: src/main/java/com/backbase/ct/bbfuel/healthcheck/HealthCheck.java [hunk 2]
```diff
@@ -13,7 +13,7 @@
 public class HealthCheck {
 
     private static final String ACTUATOR = "/actuator";
-    private static final String INFO = "/info";
+    private static final String HEALTH = "/health";
```
Comment (richard-blank) [inline src/main/java/com/backbase/ct/bbfuel/healthcheck/HealthCheck.java]: there is no added value for having this split into 2 separate constants (it is private so you can always refactor later when necessary)

File: jenkins/bb-fuel-minimal.groovy (hunk 1 above)
Comment (anilkusaksiz) [reply jenkins/bb-fuel-minimal.groovy]: Done.

File: src/main/java/com/backbase/ct/bbfuel/healthcheck/HealthCheck.java (hunk 2 above)
Comment (anilkusaksiz) [reply src/main/java/com/backbase/ct/bbfuel/healthcheck/HealthCheck.java]: Done.

**Pull request merged**